lib/config/Configuration.js

```diff
@@ -0,0 +1,91 @@
+import path from "node:path";
+import os from "node:os";
+
+/**
+ *
+ *
+ * @public
+ * @class
+ * @alias @ui5/project/config/Configuration
+ */
+export class Configuration {
```

Comment:
Remove export. Class is already the default export. This should also resolve the issue of missing methods in JSDoc.

Reply:
Done